scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -527,17 +683,11 @@ public String toString() {
     static class UserInfo {
         private String login;
         private String password;
-        private String[] groups;
+        private Collection<String> groups = Collections.EMPTY_LIST;
```

Comment:
Yes, but as it does not show any warning or compilation error and the Collections.emptyList() simply return Collections.EMPTY_LIST I don't see the point. But whatever ...